h_transport_materials/property_database/eurofer_97/aiello_2002/diffusivity.csv

```diff
@@ -0,0 +1,9 @@
+H,,D,
+X,Y,X,Y
+1.4897367831971764,1.014822110423748e-8,1.3845359845701275,1.3171516463087808e-8
+1.7462244974098888,7.683440792444838e-9,1.4884343789498915,1.0917936060893002e-8
+1.9131493084369118,3.0398387502314655e-9,1.6051374873094726,9.578367714956876e-9
+2.118639756341872,8.274691574022909e-10,1.7463149421492836,5.552159777467809e-9
+,,1.9132239253469123,2.325134966043764e-9
+,,2.116077909098515,8.207927557754549e-10
+,,2.3652056826429764,1.883244836661606e-10
```

Comment:
Something is wrong here.

The plots from Aiello's paper shows diffusivities around 1e-8 m2/s
![image](https://user-images.githubusercontent.com/40028739/236527556-8be420e2-ce08-4dd3-ae4b-33fdedd0a84c.png)

![image](https://user-images.githubusercontent.com/40028739/236527608-a9870146-645a-4738-8623-9fc5cd04e0fd.png)

Reply:
![image](https://user-images.githubusercontent.com/40028739/236528662-9bbf4ed6-66f4-4bd1-a9b3-2e70558c7821.png)
Also we should add these overlapping points